rskj-core/src/main/resources/config/testnet.conf

```diff
@@ -71,3 +71,10 @@ wallet {
     enabled = false
     accounts = []
 }
+
+cache {
+    btcBlockStore {
+        depth: 50000,
```

Comment:
This is different than the default value, is it ok?

Reply:
No, I added an extra 0, good catch!

Reply:
Yes, this is why I made it configurable, to change it in testnet. I'm testing syncing fully in testnet, and its performing much better. I will try to do more testing, but we can discuss this if you want.